### ISSUE TITLE
Misleading childNodes logging behaviour

### ISSUE THREAD

**Issue by ABCbum** [brave/brave-browser#8093]
## Description 

Brave logs a misleading message on devtools. NodeList has `[div]` inside but length is 0 and doesn't have anything inside the array. I don't know if this is a default common behaviour or not but this is really confusing to me.

![image](https://user-images.githubusercontent.com/50205705/73920940-ff44a700-48f8-11ea-8427-18604bf4cbfe.png)

## Steps to Reproduce

   1. I opened `index.html` with the following code on Brave: 
```html
<html>
  <body>
     <div id="list"></div>
     <script>
        const list = document.getElementById("list");
        const node = document.createElement("div");

        list.appendChild(node);
        console.log(list.childNodes);
        list.removeChild(node);
      </script>
  </body>
</html>
```
   2. Open devtools console.

## Actual result:

Provided on Description section.

## Expected result:
NodeList array to have `div` element inside and length being 1.

## Reproduces how often: 

Easily reproduced.

## Brave version (brave://version info)

**Brave:** 1.0.0 Chromium: 78.0.3904.97 (Official Build) (64-bit)
**Revision:**  021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
**OS:**  Windows 10 OS Version 1903 (Build 18362.592)

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel? Yes.
- Can you reproduce this issue with the dev channel? Yes.
- Can you reproduce this issue with the nightly channel? Yes.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA.
- Does the issue resolve itself when disabling Brave Rewards? NA.
- Is the issue reproducible on the latest version of Chrome? Yes.

**Comment by rebron** [comment]
@ABCbum Is this still an issue on 1.5.115? We've had quite a few chromium updates since then.

**Comment by ABCbum** [comment]
Hi @rebron , i can confirm that this is **still a bug** in current version (Version **1.5.115** Chromium: 80.0.3987.149 (Official Build) (64-bit)). It is fine with the initial load but if i reload then the bug still comes back.

![image](https://user-images.githubusercontent.com/50205705/77815392-4840fc80-70ed-11ea-956a-fbdf4e6a6603.png)

Thanks!!

**Comment by kjozwiak** [comment]
> Is the issue reproducible on the latest version of Chrome? Yes.

@ABCbum assuming it's still an issue in the last Chrome as well? Sounds like you also reproduced it on Chrome originally as per the above.

**Comment by ABCbum** [comment]
> @ABCbum assuming it's still an issue in the last Chrome as well?

@kjozwiak , yes i **can** reproduce it on Chrome (Version 81.0.4044.83 (Official Build) beta (64-bit)) the behaviour is exactly the same, fine with the initial load but reload then it's wrong. Thanks!

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
